.gitignore

```diff
@@ -13,3 +13,4 @@
 
 # Appraisal gemfile.lock
 /gemfiles/*.gemfile.lock
+out
```

Comment:
The tests generate some files in that directory for the embedded Rails test applications (e.g. Rails log files + Gemfile.lock). I did a quick investigation to see if it could easily be moved to something under `tmp/` but couldn't find where it was configured and decided to punt.